### ISSUE TITLE
Can change avatar photo

### ISSUE THREAD

**Issue by Plasmmer** [AnthyG/ThunderWave#47]

**Comment by AnthyG** [comment]
Reference to #23

**Comment by Plasmmer** [comment]
I'm suggesting a feature for now, while we can't get avatars from ZeroMe.

**Comment by AnthyG** [comment]
As good as done 🎉

**Comment by AnthyG** [comment]
... Apparently I'm to slow, and have to research a little bit, as I want the image to be cropped.. 😅

**Comment by Plasmmer** [comment]
Its better to prompt user to crop his own image.

**Comment by AnthyG** [comment]
Check out issue [#23](https://github.com/AnthyG/ThunderWave/issues/23#issuecomment-293694621) for some news!

**Comment by Plasmmer** [comment]
Please see my messages in the chat.

**Comment by AnthyG** [comment]
I have :smiley:

**Comment by Plasmmer** [comment]
?

**Comment by AnthyG** [comment]
What do I have to do?

**Comment by Plasmmer** [comment]
Please reply me in the chat.

**Comment by AnthyG** [comment]
I did?!

**Comment by Plasmmer** [comment]
I can't receive your messages, from both proxy and local.

**Comment by AnthyG** [comment]
What!?
Wow.
I hope #26 isn't true 😢

**Comment by AnthyG** [comment]
I deleted some messages in my `data.json`, are you now able to see my messages 🤔

**Comment by Plasmmer** [comment]
No. Try removing the domain from content.json (the domain isn't yet propagated in DNS).

**Comment by Plasmmer** [comment]
Oh, now the domain is propagated.

**Comment by AnthyG** [comment]
And are you able to see my messages?

**Comment by Plasmmer** [comment]
No.

**Comment by AnthyG** [comment]
That is very disturbing >:/

**Comment by Plasmmer** [comment]
Also locally, isn't working.

**Comment by AnthyG** [comment]
We should maybe open up a new issue on this 😆

# #48

**Comment by AnthyG** [comment]
I will reopen this issue, as I am currently not really able to manage getting a avatar from ZeroMe as suggested in #23

**Comment by AnthyG** [comment]
Somewhat added with commit 7714e53e892eb97f5088b6f77b6c91d211084903

**Comment by AnthyG** [comment]
For updates on this, please see https://github.com/AnthyG/ThunderWave/issues/23#issuecomment-316831488

**Comment by AnthyG** [comment]
And once again, for updates on this, please see https://github.com/AnthyG/ThunderWave/issues/23#issuecomment-317186086
